src/App.js

```diff
@@ -4,13 +4,15 @@ import { Routes, Route } from "react-router-dom";
 import Navbar from "./components/Navbar/Navbar";
 import Footer from "./components/Footer/Footer";
 import HomePage from "./pages/HomePage/HomePage";
+import ProductPage from "./pages/ProductPage/ProductPage";
 
```

Comment:
suggestion: you can use absolute import here ,  It will reduce unnecessarily lines of code and  improve your code structure , and helps in understanding the path of files .